Create ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md (#2712)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md

---
date: ""

> title: Alarms list widget can now use date range filtering.

Reviewer: ```suggestion
title: Alarms list widget now offers date range filtering
```

> title: UI as a alarms widget user i want to filter alarms by date

Author: The Claude AI suggested to use the following title:
```suggestion
title: Alarms widget allows filtering alarms by date
```

> title: UI as a alarms widget user i want to filter alarms by date

Reviewer 2: ```suggestion
title: Alarms list widget can now use date range filtering.
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60797
version: 1021.23.0
---

> UI as a alarms widget user i want to filter alarms by date

Author: The Claude AI suggested to use the following body:
```suggestion
The alarms widget in the Cumulocity IoT user interface previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in, making it easier to find relevant alarms and improving the overall user experience when working with alarms in the Cumulocity IoT application.
```

> UI as a alarms widget user i want to filter alarms by date

Reviewer 2: ```suggestion
The alarms list widget in the Cumulocity IoT user interface previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in. Users can select the date range either specifically for the widget or bind the widget to the dashboard time context.
```

> The alarms list widget in the Cumulocity IoT user interface previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in. Users can select the date range either specifically for the widget or bind the widget to the dashboard time context.

Reviewer: ```suggestion
The "Alarm list" widget previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in. Users can select the date range either specifically for the widget or bind the widget to the dashboard time context.
```
